### ISSUE TITLE
golang pprof does not work yet

### ISSUE THREAD

**Issue by cooperkuo** [gperftools/gperftools#1133]
As far as I know, perl pprof is deprecated in favor of golang, but golang pprof does not support stack traces symbolization yet. (The [pprof flag '--symbols' passed to execlp](https://github.com/gperftools/gperftools/blob/master/src/symbolize.cc#L211-L212) does not work in golang pprof.)

I was mislead by README, it recommends people to use golang pprof but sadly it does not run acutally. Is it better to remind people to use the perl pprof in README until golang pprof can work?

**Comment by alk** [comment]
yes, and they actually refuse to add support for this. Reasonably saying that they're profiling too, not symbolizing.

So for --symbols we'll have to keep perl program, or have somewhat reimplement just --symbols part in c++.
